actor/sharedaction/is_org_targeted_test.go

```diff
@@ -0,0 +1,39 @@
+package sharedaction_test
+
+import (
+	. "code.cloudfoundry.org/cli/actor/sharedaction"
+	"code.cloudfoundry.org/cli/actor/sharedaction/sharedactionfakes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IsOrgTargeted", func() {
+	var (
+		actor      *Actor
+		binaryName string
+		fakeConfig *sharedactionfakes.FakeConfig
+	)
+
+	BeforeEach(func() {
+		binaryName = "faceman"
+		fakeConfig = new(sharedactionfakes.FakeConfig)
+		fakeConfig.BinaryNameReturns(binaryName)
+		actor = NewActor(fakeConfig)
+	})
+
+	When("the config has an org targeted", func() {
+		It("returns true", func() {
+			fakeConfig.HasTargetedOrganizationReturns(true)
```

Comment:
I think this reads a little cleaner in a BeforeEach (i.e., the style of the IsLoggedIn test).

Same goes for the other test in the file and for the tests in IsSpaceTargeted.